cmd/skr/main.go

```diff
@@ -89,7 +90,7 @@ func main() {
 
 	EncodedUvmInformation, err := common.GetUvmInformation() // from the env.
 	if err != nil {
-		logrus.Fatalf("Failed to extract UVM_* environment variables: %s", err.Error())
+		logrus.Warnf("Failed to extract UVM_* environment variables: %s", err.Error())

```

Comment:
Not sure if I made this warn or not, but let's update to Info.  In the AASP case, this is expected content.